src/style.css

```
@@ -0,0 +1,70 @@
+* {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.nav {
+  display: flex;
+}
+
+.logo {
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-around;
```

```suggestion
  justify-content: space-between;
```
space-between would be more suitable here